src/components/Schedule/Schedule.js

```diff
@@ -6,9 +6,35 @@ import { TimelineColumn } from './Timeline'
 import { TagLegend } from './Tag'
 import Event from './Event'
 
+const ScrollableContainer = styled.div`
+  overflow-x: scroll;
+  overflow-y: hidden;
+  transform: rotateX(180deg);
```

Comment:
Nit: correct me if I'm wrong, but the rotation on line 12 is so that the scrollbar shows up on the top of the container and not the bottom? If this is the case maybe we can leave a quick comment on this above the start of this component

Reply:
yep that's right - i wasn't able to find a way to make the scroll bar on top other than doing the rotation. i'll leave a comment :)